src/scripts/components/story-content/story-content.styl

```diff
@@ -15,10 +16,18 @@
     font-family: NotesEsa
     line-height: emCalc(16px)
 
-  img
-    width: 50%
+  img, em
```

Comment:
this is a bit confusing, most of the properties have nothing to do with the img. could you split these up, so that both together only get the properties they need. Like
```
img, em
  ...
img
  ...
em
  ...
```